Sources/ArcGISToolkit/Components/Augmented Reality/CalibrationView.swift

```diff
@@ -0,0 +1,178 @@
+                        }
+                        headingTimer = timer
+                        // Add the timer to the main run loop.
+                        RunLoop.main.add(timer, forMode: .default)
+                    }
+                    
+                    Button {
+                        let heading = viewModel.cameraController.originCamera.heading + 1
+                        updateHeading(to: heading)
+                    } label: {
+                        Image(systemName: "plus")
+                            .imageScale(.large)
+                    }
+                }
+                VStack {
+                    Text("Elevation: \(viewModel.calibrationElevation?.rounded(.towardZero) ?? viewModel.cameraController.originCamera.location.z?.rounded(.towardZero) ?? 0, format: .number)")
+                    HStack {
+                        Button {
+                            updateElevation(by: -1)
+                        } label: {
+                            Image(systemName: "minus")
+                                .imageScale(.large)
+                        }
+                        Slider(value: $elevationSliderValue, in: -20...20) { editingChanged in
+                            if !editingChanged {
+                                elevationTimer?.invalidate()
+                                elevationTimer = nil
+                                elevationSliderValue = 0.0
+                            }
+                        }
+                        .onChange(of: elevationSliderValue) { elevation in
+                            guard elevationTimer == nil else { return }
+                            // Create a timer which rotates the camera when fired.
+                            let timer = Timer(timeInterval: 0.1, repeats: true) { _ in
+                                updateElevation(by: joystickElevationDelta)
+                            }
+                            elevationTimer = timer
+                            // Add the timer to the main run loop.
+                            RunLoop.main.add(timer, forMode: .default)
+                        }
+                        Button {
+                            updateElevation(by: 1)
+                        } label: {
+                            Image(systemName: "plus")
+                                .imageScale(.large)
+                        }
+                    }
+                }
+            }
+            .onDisappear {
+                guard !viewModel.isCalibrating else { return }
+                viewModel.setBasemapOpacity(0)
+            }
+            .overlay(alignment: .topTrailing) {
+                HStack {
+                    Spacer()
+                    Button {
+                        viewModel.isCalibrating = false
+                    } label: {
+                        Image(systemName: "xmark.circle.fill")
+                            .font(.system(size: 20))
```

Comment:
```suggestion
```
The font size is specified here to make the image larger, similar to the size of the x icon in the "find my" app. I can remove this line if the default size looks good.

Reply:
We shouldn't use font sizes directly. We should always use dynamic fonts. Also the imageScale that I showed should change the size of the image.

Reply:
Good point. The image size is too small using the  `.large` image scale. However, using the `.title3` font improves the icon size.

Reply:
Actually, I guess if we are using the font size to define the size of the image then it should be hard-coded because we don't want that changing with the users font size settings.

Maybe a better way to do that though is to just set the frame width/height of the image.

Reply:
Done. The image size is now set with the frame width and height.